docs/gambit/mutation-verifier.md

```diff
@@ -18,19 +18,21 @@ Installation instructions can be found [here](https://www.java.com/en/download/h
 
 ## Running the Mutation Verifier
 
-- Example:
+You will need to update your `certora-cli-alpha-master` installation using `pip` to get the relevant
+dependencies. Then you can run Gambit like so:
+
```

Comment:
I think since we're already having them install through the `certora-cli-alpha-master`, we shouldn't direct them directly to the Java page.  Instead, we should direct them to the installation instructions for the Prover.  Something like
````
To use the mutation verifier, you will need to {ref}`install the Certora Prover and its dependencies <installation>`.  Mutation verification currently requires the `alpha-master` version of the Certora CLI.  To install it, run

```sh
pip install certora-cli-alpha-master
```

If you already have `certora-cli-alpha-master` installed and the `mutationTest` command is not available, you may need to update to a newer version by running

```sh
pip install --upgrade certora-cli-alpha-master
```

````